src/libs/OptionsListUtils.js

```diff
@@ -1358,7 +1359,22 @@ function getOptions(
     const searchValue = parsedPhoneNumber.possible ? parsedPhoneNumber.number.e164 : searchInputValue.toLowerCase();
 
     // Filter out all the reports that shouldn't be displayed
-    const filteredReports = _.filter(reports, (report) => ReportUtils.shouldReportBeInOptionList(report, Navigation.getTopmostReportId(), false, betas, policies));
+    const filteredReports = _.filter(reports, (report) => {
+        const {parentReportID, parentReportActionID} = report;
+        const canGetParentReport = parentReportID && parentReportActionID && allReportActions;
+        const parentReportAction = canGetParentReport ? lodashGet(allReportActions, [parentReportID, parentReportActionID], {}) : {};
```

Comment:
```suggestion
        const parentReportAction = ReportActionUtils.getParentReportAction(report);
```

Reply:
`ReportActionUtils.getParentReportAction()` is deprecated, which is why we're getting in manually here.

![CleanShot 2024-01-12 at 13 26 55@2x](https://github.com/Expensify/App/assets/22041394/213b276e-5c5e-4cd4-a3bb-f495df54ff45)

Reply:
ok but this is still being used in many places.
Btw, I reported here because of [this crash](https://github.com/Expensify/App/pull/31448#issuecomment-1888004929)